buildscripts/gateway-tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -e
+
+export SERVER_ENDPOINT=127.0.0.1:24240
+export ENABLE_HTTPS=0
+export ACCESS_KEY=minio
+export SECRET_KEY=minio123
+export MINIO_ACCESS_KEY=minio
+export MINIO_SECRET_KEY=minio123
+export AWS_ACCESS_KEY_ID=minio
```

Comment:
You don't need to set this if your backend and front-end creds are same for s3 gateway

Reply:
But this should be set inside `awscli.sh` code already @wlan0 you don't need to do this here.